src/workloads/scale/MixedWorkloadsGennyStress.yml

```diff
@@ -148,7 +148,7 @@ Actors:
   Phases:
   - Repeat: 1
     BatchSize: 100
-    Threads: 8
+    Threads: 1
```

Comment:
I'm a bit confused why this is necessary if the workload is correctly loading documents.  Should we add some documentation with this change about difference between the Threads setting on phases and Actors? Will this reduce the speed of the load phase?

Reply:
I misread my own code again. I'm going to push an update, so they are both 8. This workload has 512 collections, so 8 threads can insert into 64 collections a piece. 

What i think is happening today is 1 thread is pushing documents into 64 collections, rather than 8 threads pushing into 512 collections total.